zb-db/src/main/java/io/camunda/zeebe/db/impl/rocksdb/transaction/ZeebeTransactionDb.java

```diff
@@ -85,7 +107,14 @@ ZeebeTransactionDb<ColumnFamilyNames> openTransactionalDb(
     final OptimisticTransactionDB optimisticTransactionDB =
         OptimisticTransactionDB.open(options, path);
     closables.add(optimisticTransactionDB);
-    final var defaultColumnFamilyHandle = optimisticTransactionDB.getDefaultColumnFamily();
```

Comment:
:question: Was this removed?

Reply:
No, I don't fully understand why it no longer works, but this method returns null now, regardless of whether RocksDB.loadLibrary has been called already.

Using make resolved it, but TBH I don't fully get why it didn't work without anymore.

Reply:
Hmm, okay finally had another look at this. The [RocksJava Basics](https://github.com/facebook/rocksdb/wiki/RocksJava-Basics#opening-a-database-with-column-families) show a way to get column family handles. The example is specific to opening the database with multiple column families, but I think this approach might work for a single column family as well. This same syntax is available for the [optimistic transactional database](https://github.com/facebook/rocksdb/blob/b46f5707c4092a2972941aad3308464887857157/java/src/main/java/org/rocksdb/OptimisticTransactionDB.java#L51-L69) as well. I think I'll give this a try instead.